lib/Zonemaster/Backend/RPCAPI.pm

```diff
@@ -191,65 +191,55 @@ sub get_data_from_parent_zone {
 
 
 sub _check_domain {
-    my ( $self, $dn, $type ) = @_;
+    my ( $self, $domain, $type ) = @_;
```

Comment:
I think it would help if the API of this method is defined. Especially spell out what "$type" is for.